packages/lib-classifier/src/store/subjects/Subject/Subject.spec.js

```diff
@@ -76,12 +74,6 @@ describe('Model > Subject', function () {
   })
 
   describe('Views > viewer', function () {
-    it('should return null as default', function () {
-      const store = mockStore({ project, workflow, subject: stub })
-      const subject = store.subjects.active
-      expect(subject.viewer).to.be.null()
-    })
```

Comment:
This test is removed because it was actually testing "should return null viewer for subjects with more than ten location". The `stub` subject from SubjectFactory() has 11 locations.